.github/workflows/integration.yml

```diff
@@ -0,0 +1,50 @@
+name: continuous-integration
+
+on: 
+  push:
+    branches:
+      - dev
+      - master
+    paths:
+      - 'src/**'
+      # - 'test/**'
+      - 'build-libs.sh'
+      - 'Dockerfile'
+      - 'Makefile.essentiajs'
+      - 'rollup*.js'
+      - 'tsconfig.json'
+      - '!docs/**'
+      - '!examples/**'
+  pull_request:
+    branches:
+      - dev
+      - master
+    paths:
+      - 'src/**'
+      # - 'test/**'
+      - 'build-libs.sh'
+      - 'Dockerfile'
+      - 'Makefile.essentiajs'
+      - 'rollup*.js'
+      - 'tsconfig.json'
+      - '!docs/**'
```

Comment:
same comment as above

Reply:
yep, again, the `!` excludes those paths